chain/network/src/peer_manager/tests.rs

```diff
@@ -326,7 +326,7 @@ async fn accounts_data_rate_limiting() {
     let mut pms = vec![];
     for _ in 0..n * m {
         let mut cfg = chain.make_config(rng);
-        cfg.accounts_data_broadcast_rate_limit = demux::RateLimit { qps: 0.5, burst: 1 };
+        cfg.accounts_data_broadcast_rate_limit = demux::RateLimit { qps: 1., burst: 1 };
```

Comment:
this doesn't seem to be related to thePR description

Reply:
yeah, moved to a separate PR